howto/anbox/enable-virgl.md

```diff
@@ -0,0 +1,53 @@
+With the 1.21 release Anbox Cloud has introduced a new rendering path to provide [Vulkan API](https://www.vulkan.org/) support for NVIDIA GPUs. This new stack is called VirGL as it it is based on the open source Mesa [Venus driver](https://docs.mesa3d.org/drivers/venus.html) and the [virglrenderer](https://gitlab.freedesktop.org/virgl/virglrenderer) project. You can find more details in [Rendering architecture](https://discourse.ubuntu.com/t/35129).
+
+In order to provide a migration path for existing users, the new rendering pipeline based on the VirGL stack is disabled by default in the 1.21 release of Anbox Cloud. With 1.22 it will become the default.
```

Comment:
```suggestion
In order to provide a migration path for existing users, the new rendering pipeline based on the VirGL stack is disabled by default in the 1.21 release of Anbox Cloud. With future releases, this will become the default.
```